BeeSwift/MainCoordinator.swift

```diff
@@ -1,240 +1,255 @@
+      viewContext: viewContext,
+      coordinator: self)
+    navigationController.pushViewController(goalViewController, animated: true)
+  }
+
+  func showSettings() {
+    let settingsVC = SettingsViewController(
+      currentUserManager: currentUserManager,
+      viewContext: viewContext,
+      goalManager: goalManager,
+      requestManager: requestManager,
+      coordinator: self)
+    navigationController.pushViewController(settingsVC, animated: true)
+  }
+
+  func showSignIn() {
+    let signInVC = SignInViewController(currentUserManager: currentUserManager, coordinator: self)
+    signInVC.modalPresentationStyle = .fullScreen
+    navigationController.present(signInVC, animated: true)
+  }
+
+  func showTimerForGoal(_ goal: Goal) {
+    let controller = TimerViewController(goal: goal, requestManager: requestManager)
+    controller.modalPresentationStyle = .fullScreen
+    navigationController.present(controller, animated: true, completion: nil)
+  }
+
+  func showChooseGallerySortAlgorithm() {
+    let controller = ChooseGoalSortViewController()
+
+    navigationController.pushViewController(
+      controller,
+      animated: true)
+  }
+
+  func showConfigureNotifications() {
+    let controller = ConfigureNotificationsViewController(
+      goalManager: goalManager,
+      viewContext: viewContext,
+      currentUserManager: currentUserManager,
+      requestManager: requestManager,
+      coordinator: self)
+
+    navigationController.pushViewController(
+      controller,
+      animated: true)
+  }
+
+  func showConfigureHealthKitIntegration() {
+    let controller = HealthKitConfigViewController(
+      goalManager: goalManager,
+      viewContext: viewContext,
+      healthStoreManager: ServiceLocator.healthStoreManager,
+      requestManager: requestManager,
+      coordinator: self)
+
+    navigationController.pushViewController(
+      controller,
+      animated: true)
+  }
+
+  func showRemoveHealthKitIntegrationFromGoal(_ goal: Goal) {
+    let controller = RemoveHKMetricViewController(
+      goal: goal,
+      requestManager: requestManager,
+      goalManager: goalManager)
+
+    navigationController.pushViewController(
+      controller,
+      animated: true)
+  }
+
+  func showAssociateHealthKitWithGoal(_ goal: Goal) {
+    let chooseHKMetricViewController = ChooseHKMetricViewController(
+      goal: goal,
+      healthStoreManager: healthStoreManager,
+      requestManager: requestManager,
+      coordinator: self)
+
+    navigationController.pushViewController(
+      chooseHKMetricViewController,
+      animated: true)
+  }
+
+  func showConfigureNotificationsForGoal(_ goal: Goal) {
+    let controller = EditGoalNotificationsViewController(
+      goal: goal,
+      currentUserManager: currentUserManager,
+      requestManager: requestManager,
+      goalManager: goalManager,
+      viewContext: viewContext)
+
+    navigationController.pushViewController(
+      controller,
+      animated: true)
```

Comment:
Does this really need to wrap at all?

Reply:
I thought perhaps this was affected by [linebreakbeforeeachargument](https://github.com/swiftlang/swift-format/blob/main/Documentation/Configuration.md#linebreakbeforeeachargument) being set to `false` in combination with the line being longer than 120 characters if placed horizontally but the line is 73 chars or so.
Nonetheless, I find it easier to read and more consistent with the rest of the file that such calls are split over multiple lines.